src/mongo_client.ts

```diff
@@ -427,13 +419,9 @@ export class MongoClient extends EventEmitter implements OperationParent {
   static connect(url: string, callback: Callback<MongoClient>): void;
   static connect(url: string, options: MongoClientOptions): Promise<MongoClient>;
   static connect(url: string, options: MongoClientOptions, callback: Callback<MongoClient>): void;
-  static connect(
-    url: string,
-    options?: MongoClientOptions | Callback<MongoClient>,
-    callback?: Callback<MongoClient>
-  ): Promise<MongoClient> | void {
-    if (typeof options === 'function') (callback = options), (options = {});
-    options = options || {};
+  static connect(url: string, ...args: any[]): Promise<MongoClient> | void {
```

Comment:
This is another place for the labeled tuple style that we could use when api-extractor supports TS 4

Reply:
Reverted this, it was left over from an attempt to use the labeled tuples.